engine/src/main/java/org/entando/entando/web/common/model/Filter.java

```diff
@@ -90,6 +93,13 @@ public EntitySearchFilter getEntitySearchFilter() {
         return filter;
     }
 
+    public List<String> getFilterValues() {
```

Comment:
Since this is basically made for the OR conditions, to make it more explicit and easier to interpret without the need for comments what if we change the name to something like "getOrConditions" or "getOrValues" or something like that?